crates/cxx-qt-lib/src/core/qstring.rs

```diff
@@ -114,6 +118,15 @@ impl fmt::Debug for QString {
     }
 }
 
+impl std::ops::Add for QString {
+    type Output = Self;
+    fn add(self, other: Self) -> Self {
```

Comment:
Hmm assume you can't use `QString &QString::operator+=(const QString &other)` ? 

Guess due to the types of `Add` this is tricky, as is closer to `AddAssign` ?